pkg/types/baremetal/validation/libvirt.go

```diff
@@ -40,25 +40,45 @@ func validateInterfaces(p *baremetal.Platform, fldPath *field.Path) field.ErrorL
 // to validate if an interface is found among them
 func interfaceValidator(libvirtURI string) (func(string) error, error) {
 	// Connect to libvirt and obtain a list of interface names
+	interfaces := make(map[string]struct{})
+	var exists = struct{}{}
 	conn, err := libvirt.NewConnect(libvirtURI)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not connect to libvirt")
 	}
 
-	interfaces, err := conn.ListAllInterfaces(libvirt.CONNECT_LIST_INTERFACES_ACTIVE)
+	networks, err := conn.ListAllNetworks(libvirt.CONNECT_LIST_NETWORKS_ACTIVE)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not list libvirt networks")
+	}
+	for _, network := range networks {
+		networkName, err := network.GetName()
+		if err == nil {
+			bridgeName, err := network.GetBridgeName()
+			if err == nil && bridgeName == networkName {
```

Comment:
What happens if these are different? Like, in the case of the 'default' network, the bridge might be 'virbr0' right?

Reply:
we purposely skip them, because bootstrap vm would fail to launch since the network snippet is something like this

```
    <interface type='bridge'>
      <source bridge='mybridge'/>
    </interface>
```